readme.txt

```diff
@@ -0,0 +1,27 @@
+=== Wikipedia Preview ===
+Contributors: Wikimedia Foundation
+Donate link: 
+Tags: wikipedia, facts, popup, card, wiki
+Stable tag: trunk
+License: MIT
+License URI: https://github.com/wikimedia/wikipedia-preview/blob/main/LICENSE
```

Comment:
Let's please confirm license info

Reply:
I'm 90% sure MIT is OK. Legal will confirm during their review.

Reply:
should this be the license of `wikipediapreview-wordpress` or `wikipedia-preview`?

Reply:
Adding here the WordPress license guideline just for reference: 

https://developer.wordpress.org/plugins/wordpress-org/detailed-plugin-guidelines/#1-plugins-must-be-compatible-with-the-gnu-general-public-license

> should this be the license of wikipediapreview-wordpress or wikipedia-preview?
I asked myself the same question

Reply:
It's the license of the plugin but both have to be compatible since Wikipedia Preview is bundled with it. MIT across the board is a safe bet.